tests/test_integration/test_operations.py

```diff
@@ -0,0 +1,21 @@
+"""Integration tests for operations in the service."""
+
+import pytest
+import requests
+
+SUCCESS_STATUS_CODE = 200
```

Comment:
Instead of this please use the already present `http` module, ie:
```python
from http import HTTPStatus

HTTPStatus.OK # This would give you 200
```